src/components/Menu.jsx

```diff
@@ -12,6 +12,8 @@ const propTypes = {
   renderItemLabel: PropTypes.func,
   value: PropTypes.string,
   onSelect: PropTypes.func,
+  shouldTransferFocus: PropTypes.bool,
+  notifyBeforeFocusWrap: PropTypes.func,
 };
```

Comment:
Thanks for clarifying the naming convention of callbacks. So, what you're saying is that the behavior of the keyDown handler in the Menu component can be completely overridden by the parent via the callback if the callback exists?